lms/static/scripts/frontend_apps/components/EmailPreferencesApp.tsx

```diff
@@ -25,13 +51,19 @@ export default function EmailPreferencesApp() {
           className="h-10"
         />
       </div>
+      <div className="absolute top-20 w-full flex justify-center">
+        <div className="w-80">
+          <ToastMessages
+            messages={toastMessages}
+            onMessageDismiss={dismissToastMessage}
+          />
```

Comment:
I took the handling of the message off the `EmailPreferences` component, and into `EmailPreferencesApp`.

I think it makes more sense to handle it this way, because in other contexts this might be handled differently, so doesn't really make sense to embed `ToastMessages` inside `EmailPreferences`.